GameServer/managers/playermanager/FriendsManager.cs

```diff
@@ -123,21 +123,25 @@ public void RemovePlayerFriendsListFromCache(GamePlayer Player)
 			if (Player == null)
 				throw new ArgumentNullException("Player");
 
-			string[] friends;
-			if (!PlayersFriendsListsCache.TryRemove(Player, out friends))
-			{
-				if (log.IsWarnEnabled)
-					log.WarnFormat("Gameplayer ({0}) was not registered in Friends Manager Cache while trying to remove!", Player);
-			}
-			
-			FriendStatus[] offlineFriends;
-			if (!PlayersFriendsStatusCache.TryRemove(Player, out offlineFriends))
+			string[] friends = null;
+			//<**loki**>
+			if (friends != null)
```

Comment:
I think this is the issue with unit tests.

Reply:
I did this to avoid ugly console spam on kicking player only. Ill correct